docs/src/main/asciidoc/_elytron/Using_WildFly_Elytron_with_WildFly.adoc

```diff
@@ -12,23 +12,6 @@ box Elytron components for securing the management interfaces in the
 link:Elytron_Subsystem.html#src-557147_ElytronSubsystem-DefaultManagementAuthenticationConfiguration[Default
 Management Authentication Configuration] section.
 
-[[securing-applications]]
-=== Securing Applications
-
```

Comment:
This section may need some alternative - although we have removed the application-http-authentication resource from the Elytron subsystem an application-security-domain resource will still be needed in the Undertow subsystem to reference the ApplicationDomain instead.

Reply:
+1

Reply:
Added section to reference ApplicationDomain